### ISSUE TITLE
Isnegate condition written wrong

### ISSUE THREAD

**Issue by sperdha123** [Intacct/intacct-sdk-php#150]
Negate condition is written in a wrong way in PHP SDK.
Negate will not work until and unless fixed by below solution.

NOT IN Should be there after the field not before the field
Example: 
`RECORDNO  NOT IN ('89803','42409')`
Instead of 
`NOT  RECORDNO  IN ('89803','42409')`

**Current Code:**
```php
public function __toString(): string
    {
        $clause = '';
        if ($this->isNegate() === true) {
            $clause = 'NOT ';
        }
        $pieces = [];
        foreach ($this->getValue() as $value) {
            $pieces[] = "'" . addcslashes($value, "'") . "'";
        }
        $clause .= $this->getField() . " IN (" . implode(',', $pieces) . ")";

        return $clause;
    }
```
**Working Expected Code:**
```php
public function __toString(): string
    {
        $clause = '';
        $notClause = '';
        if ($this->isNegate() === true) {
            $notClause = ' NOT';
        }
        $pieces = [];
        foreach ($this->getValue() as $value) {
            $pieces[] = "'" . addcslashes($value, "'") . "'";
        }
        $clause = $this->getField() . **$notClause** . " IN (" . implode(',', $pieces) . ")";

        return $clause;
    }
```

**Comment by jimmymcpeter** [comment]
Good catch. This applies to both InArrayInteger and InArrayString classes.  You may want to be careful with your type casting with recordno being an integer.  Oddly the query works if you nest in something that does... 🤦  `<query>NOT VENDORID IN ('V0001') AND VENDORID IN ('V0002')</query>`

**Comment by intacctcoleenho** [comment]
Fixing in next release.

**Comment by intacctcoleenho** [comment]
Fixed in v2.3.0
